migrations/0001_setup_models.sql

```diff
@@ -1,3 +1,36 @@
 -- DropTable
 PRAGMA foreign_keys=off;
 PRAGMA foreign_keys=on;
+
+-- CreateTable
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "username" TEXT NOT NULL,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- CreateTable
+CREATE TABLE "Credential" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "userId" TEXT NOT NULL,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "credentialId" TEXT NOT NULL,
+    "publicKey" BLOB NOT NULL,
+    "counter" INTEGER NOT NULL DEFAULT 0,
+    CONSTRAINT "Credential_userId_fkey" FOREIGN KEY ("userId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "User_username_key" ON "User"("username");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Credential_userId_key" ON "Credential"("userId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Credential_credentialId_key" ON "Credential"("credentialId");
+
+-- CreateIndex
+CREATE INDEX "Credential_credentialId_idx" ON "Credential"("credentialId");
+
+-- CreateIndex
+CREATE INDEX "Credential_userId_idx" ON "Credential"("userId");
```

Comment:
I needed to add these in order for `pnpm migrate:dev` to work (they're the same as what is in the standard starter). Otherwise the second migration trips up since it tries to alter the User table, but the User table does not yet exist.